alerts/HighlyAvailableWorkloadIncorrectlySpread.md

```diff
@@ -0,0 +1,89 @@
+# HighlyAvailableWorkloadIncorrectlySpread
+
+## Meaning
+
+Highly available workloads with persistent storage are incorrectly spread across
+multiple nodes. In high availability topology, all workloads with multiple
+replicas should be spread across multiple nodes to avoid having a single point
+of failure. However, workload with persistent storage are more complex to
+schedule since persistent volumes may be bound to an availability zone, so they
+need to be spread manually before any scheduling constraints are put in place.
+
+## Impact
+
+The cluster isn't fully highly-available since workloads with persistent
```

Comment:
```suggestion
The cluster isn't fully highly available since workloads with persistent
```
Further up we used `highly available`. I think no hyphen is correct, but at least lets be consistent.